src/cow-react/common/pure/CurrencyInputPanel/index.tsx

```diff
@@ -78,6 +80,8 @@ export function CurrencyInputPanel(props: CurrencyInputPanelProps) {
   return (
     <>
       <styledEl.Wrapper id={id} className={className} withReceiveAmountInfo={!!receiveAmountInfo}>
+        {topLabel && <styledEl.CurrencyTopLabel>{topLabel}</styledEl.CurrencyTopLabel>}
```

Comment:
<Trans>{topLabel}</Trans>

Reply:
Are you suggesting to pass all this as a prop instead of having the html and the condition here?

Reply:
Sorry, github swallowed my code :) 
I mean, we should add `<Trans>` here